frontend/src/screens/CreateNewAccountScreen.js

```diff
@@ -9,44 +9,99 @@ import buttonStyle from '../styles/BottomButtonStyle';
 import navigationPropTypes from '../helpers/navigationPropTypes';
 
 
-export default function CreateNewAccountScreen({ navigation, token, postNewAccount }) {
+export default function CreateNewAccountScreen({
+  navigation, token, postNewAccount, status,
+}) {
   const [accountName, onChangeText] = React.useState('');
-
-  const handleChange = ({ nativeEvent }) => {
-    onChangeText(nativeEvent.text);
+  const handleChange = (value) => {
+    onChangeText(value);
   };
 
+
   const handlePress = () => {
     postNewAccount(accountName, token);
-    navigation.navigate('Accounts');
   };
 
-  return (
-    <KeyboardAvoidingView style={{ flex: 1 }} behavior="padding" enabled>
-      <View style={{ flex: 0.5 }}>
-        <Text style={styles.text}>Name of deposit account</Text>
-      </View>
-      <View style={{ flex: 0.5 }}>
-        <Item style={{ margin: 10 }}>
-          <Input
-            name="accountName"
-            onChange={handleChange}
-            value={accountName}
-          />
-        </Item>
-      </View>
-      <View style={{ flex: 6, flexDirection: 'column-reverse', alignItems: 'flex-end' }}>
+  function renderGoBackButton() {
```

Comment:
if you create a component, separate it from the current one
```suggestion
  function goBackButton() {
```